src/app/versioned-folder/versioned-folder-detail/versioned-folder-detail.component.html

```diff
@@ -0,0 +1,270 @@
+          <button
+            mat-menu-item
+            type="button"
+            [matMenuTriggerFor]="userActionsDelete"
+            *ngIf="access.showSoftDelete || access.showPermanentDelete"
+            class="warning"
+          >
+            <i class="far fa-trash-alt warning"></i> Delete
+          </button>
+        </mat-menu>
+        <!-- End of first level menu -->
+
+        <!--Second level sub Menu-->
+        <mat-menu #userActionsDelete="matMenu">
+          <button
+            mat-menu-item
+            (click)="askForSoftDelete()"
+            *ngIf="access.showSoftDelete && !detail.deleted"
+          >
+            Mark as deleted
+          </button>
+          <button
+            mat-menu-item
+            (click)="askForPermanentDelete()"
+            *ngIf="access.showPermanentDelete"
+          >
+            Delete <span class="warning">permanently</span>
+          </button>
+        </mat-menu>
+        <!-- End of Second level sub Menu-->
+
+        <div
+          *ngIf="access.showPermission"
+          style="position: relative; float: right; margin-right: 8px"
+          [hidden]="editor.isEditing"
+        >
+          <button
+            mat-icon-button
+            color="primary"
+            type="button"
+            class="paddingless"
+            (click)="showSecurityDialog()"
+            matTooltip="User & Group Access"
+            aria-label="User & Group Access"
+          >
+            <i class="fas fa-user-lock"></i>
+          </button>
+        </div>
+
+        <div
+          style="
+            position: relative;
+            float: right;
+            margin-right: 8px;
+            margin-left: 8px;
+          "
+          [hidden]="editor.isEditing"
+        >
+          <button
+            mat-icon-button
+            color="primary"
+            type="button"
+            class="paddingless"
+            (click)="toggleShowSearch()"
+            [disabled]="processing"
+            matTooltip="Search"
+            aria-label="Search"
+          >
+            <i class="fas fa-search"></i>
+          </button>
+        </div>
+      </div>
+
+      <div *ngIf="detail">
+        <div class="data-model-header">
+          <h4
+            class="inline-block marginless"
+            [ngClass]="{ deletedDataModelTitle: detail?.deleted }"
+          >
+            <span class="dataModelTypeIcon">
+              <i
+                matTooltip="Versioned Folder"
+                class="fas fa-project-diagram"
+              ></i>
+            </span>
+            <span
+              *ngIf="!editor.isEditing"
+              class="dataModelDetailsLabel"
+              style="font-weight: 500"
+              >{{ detail.label }}</span
+            >
+            <mat-form-field
+              *ngIf="editor.isEditing"
+              appearance="outline"
+              style="width: 100%"
+            >
+              <input
+                matInput
```

Comment:
Updated to the latest controls

Reply:
Changed this to the updated `<mdm-inline-text-edit>` control